### ISSUE TITLE
Unavailble Quat(euler) constructor

### ISSUE THREAD

**Issue by Lisapple** [godotengine/godot-docs#2421]
The `Quat(euler: Vector3)` is proposed for GDscript in the 3.1.1 documentation but is not available (not even in the engine C++ source code).

```
Quat(Vector3()) # Error: Invalid arguments for 'Quat' contructor
```

**Comment by tomglenn** [comment]
Just wanted to add that this issue is still present.

**Comment by NathanLovato** [comment]
I just tested and it seems to be fixed in Godot 3.2. `Quat(Vector3.ZERO)` doesn't lead to an error.
